app/views/devise/sessions/new.html.erb

```diff
@@ -5,11 +5,11 @@
 <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
   <fieldset>
     <div class="form-group">
-      <%= f.text_field :login, class: 'form-control', placeholder: t('.login'), autofocus: true %>
+      <%= f.text_field :login, class: 'form-control', placeholder: 'login', autofocus: true %> <%= t('.login') %>
```

Comment:
* The placeholder isn't localized in that first part.  We need to maintain i18n on this form.

* Why add text after that?